web/web.go

```diff
@@ -263,7 +265,20 @@ func (h *Handler) statush(w http.ResponseWriter, r *http.Request) {
 	h.status.mu.RLock()
 	defer h.status.mu.RUnlock()
 
-	h.executeTemplate(w, "status", h.status)
+	h.executeTemplate(w, "status", struct {
+		Status *PrometheusStatus
+		Info   map[string]string
+	}{
+		Status: h.status,
+		Info:   version.Map,
+	})
+}
+
+func (h *Handler) version(w http.ResponseWriter, r *http.Request) {
```

Comment:
I was 50:50 on this. Checked other projects with multiple API versions and have seen the version endpoint outside of the API scope. One can argue that this endpoint should be super-stable so no matter what we do to APIs (depracting a version for example), clients can always verify a version change.